boudicca.base/common-model/src/main/kotlin/base/boudicca/model/structured/VariantConstants.kt

```diff
@@ -17,6 +17,8 @@ object VariantConstants {
         const val NUMBER_FORMAT_NAME = "number"
         const val DATE_FORMAT_NAME = "date"
         const val LIST_FORMAT_NAME = "list"
+        const val ENUM_FORMAT_NAME = "enum"
```

Comment:
what does format=enum mean? what enum?

Reply:
didn't we also discuss enums and json as formats for variants when we originally envisioned these variants?
Mainly we need to find a generic way to enter enums

Reply:
yes we did discuss it, and we have some in our semantic conventions as well. but i am not sure what marking as a format brings us. maybe just for metadata purposes, so someone knows somewhere is a enum defined? but i am really not sure

Reply:
i created a separate ticket for it. added them now for completeness, but left them empty as to indicate use of text format.